src/NuGetGallery/Infrastructure/Lucene/LuceneSearchService.cs

```diff
@@ -27,6 +27,8 @@ public class LuceneSearchService : ISearchService
 
         public bool ContainsAllVersions { get { return false; } }
 
+        public bool SupportsAdvancedSearch { get { return false; } }
```

Comment:
You can shorten this to:

```suggestion
        public bool SupportsAdvancedSearch => false;
```

The arrow syntax is relatively new, hence why `ContainsAllVersions` doesn't use it. You can update that property too :)